docs/learn/learn-comparisons-avalanche.md

```diff
@@ -0,0 +1,112 @@
+
+- **Staking amount:** This value defines the minimal stake required to be placed as
+bond before participating in the system.
+- **Minimum staking time for a node:** The minimal amount of time required for a node to stake into the system.
+- **Maximum staking time for a node:** The maximal amount of time a node can stake.
+- **Minting rate:** Reward rate function, also referred to as minting rate, determines the reward a
+participant can claim as a function of their staking amount given some number of x publicly disclosed nodes
+under its ownership, over a period of t consecutive *minimal staking time* timeframes, such that t*minimal staking time* ≤ *maximum staking time*.
+- **Transaction fee amount:** The fee structure, which is a set of governable fees parameters that specify costs to various transactions.
+
+Limiting the governance functionality is a design choice to increase predictability and safety.
+
+Polkadot's governance mechanism has been in production from the very beginning and was used to slowly release functionality and decentralize the initial network. It is also not limited to a few parameters and in fact, the whole runtime is subject to change via protocol making Polkadot a meta-protocol.
+
+## Upgrades
+
+The upgrades to Avalanche are administered by the protocol developers at Avax Labs. On Polkadot, the forkless upgrades are administered and deployed through the on-chain governance.
+
+## In the physical realm
+
+Ava Labs, founded by Emin Gün Sirer is based in Brooklyn, NY, and is currently the main organization behind the development of Avalanche consensus and the Avalanche Primary Network. Emin, a professor at Cornell and known for his work in the Bitcoin ecosystem, namely around the "selfish mining" attack together with Maofan Ted Yin produced the research and ideas for Avalanche at Cornell. 
+
+## Conclusion
+
+Avalanche has made some design decisions that allow for an improved smart-contract development environment in which protocol engineers can have the freedom to create their own blockchains and include them in the Avalanche ecosystem via subnets. The trade-offs are that the autonomy of design is limited and blockchains have to buy into the design decisions of Avalanche's main chains. Subnets are not able to share the security of the main chains. However, they are able to utilize the low-transaction fee to create working applications quickly. We have even seen some Ethereum projects move to Avalanche to take advantage of the lower fees. Whereas Polkadot provides a heterogeneous model, in which blockchains have ultimate autonomy in the way they design their consensus systems, and on top of that are able to share the security of the Relay-chain, and use XCM to pass native trustless messages, instead of having to rely on bridges.
```

Comment:
> Whereas Polkadot provides a heterogeneous model, in which blockchains have ultimate autonomy in the way they design their consensus systems

Might need to rephrase this. Parachains cannot have their own consensus model for Block finality. Block production, yes. But not finality.

```suggestion
Avalanche has made some design decisions that allow for an improved smart-contract development environment in which protocol engineers can have the freedom to create their own blockchains and include them in the Avalanche ecosystem via subnets. The trade-offs are that the autonomy of design is limited and blockchains have to buy into the design decisions of Avalanche's main chains. Subnets are not able to share the security of the main chains. However, they are able to utilize the low-transaction fee to create working applications quickly. We have even seen some Ethereum projects move to Avalanche to take advantage of the lower fees. Whereas Polkadot provides a heterogeneous model, in which blockchains have ultimate autonomy in the way they design their consensus systems, and on top of that are able to share the security of the Relay-chain, and use XCM to pass native trustless messages, instead of having to rely on bridges.
```